0011-command-execution-protocol/README.md

```diff
@@ -0,0 +1,170 @@
+---
+name: Command Execution Protocol
+shortname: 11/CEP
+status: raw
+editor: Alexey Shmalko <ashmalko@kaaiot.io>
+---
+
+<!-- toc -->
+
+# Introduction
+Command Execution Protocol is an endpoint-aware 1/KP extension. It is designed to allow command execution on endpoints.
```

Comment:
Yes, first three cons are basically why I think avoiding merge conflicts is not that important.

For the fourth one, I do talk about markdown; `code` refers to markdown code.

Why is that friendlier to human eyes? I don't see much difference. For me, multi-sentence lines are more natural as that's actually how markdown is rendered—all on a single line.

To make it clear, I am not for establishing an opposite rule but to relax the current one, so that everyone can write as they want.